modules/cnv-viewing-vmi-ip-cli.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc
+
+[id="cnv-viewing-vmi-ip-cli_{context}"]
+= Viewing the IP address of a virtual machine interface in the CLI
+
+The network interface configuration is included in the `oc describe vmi` command.
+
+You can also view the IP address information by running `ip addr` on the virtual
+machine, or by running `oc get vmi -o yaml`.
```

Comment:
Where to run `oc get vmi -o yaml`? Master?

Reply:
Anywhere that has the client installed.
We have a user story for the 1.4 migration designed around covering the CLI tools, to cover things like `oc login`, but it doesn't exist here yet.